src/components/vm/consoles/vnc.jsx

```diff
@@ -171,6 +171,9 @@ class Vnc extends React.Component {
                         textDisconnect={_("Disconnect")}
                         consoleContainerId={isExpanded ? "vnc-display-container-expanded" : "vnc-display-container-minimized"}
                         resizeSession={!isExpanded}
+                        clipViewport
+                        dragViewport
+                        scaleViewport
```

Comment:
Looks like scalling and clipping don't work well together? https://github.com/novnc/noVNC/blob/master/core/rfb.js#L344